.github/actions/to-integration-tests/entrypoint.sh

```diff
@@ -0,0 +1,242 @@
+-----END RSA PRIVATE KEY-----
+" > localhost.key
+
+cat <<-EOF >cdn.conf
+{
+	"hypnotoad" : {
+		"listen" : [
+			"https://not-a-real-host.test:1?cert=$PWD/localhost.crt&key=$PWD/localhost.key&verify=0x00&ciphers=AES128-GCM-SHA256:HIGH:!RC4:!MD5:!aNULL:!EDH:!ED"
+		],
+		"user" : "trafops",
+		"group" : "trafops",
+		"heartbeat_timeout" : 20,
+		"pid_file" : "/var/run/traffic_ops.pid",
+		"workers" : 12
+	},
+	"use_ims": true,
+	"traffic_ops_golang" : {
+		"insecure": true,
+		"port" : "6443",
+		"log_location_error": "stderr",
+		"log_location_warning": "stderr",
+		"log_location_info": null,
+		"log_location_debug": null,
+		"log_location_event": null,
+		"max_db_connections": 20,
+		"db_conn_max_lifetime_seconds": 60,
+		"db_query_timeout_seconds": 20,
+		"supported_ds_metrics": [ "kbps", "tps_total", "tps_2xx", "tps_3xx", "tps_4xx", "tps_5xx" ]
+	},
+	"cors" : {
+		"access_control_allow_origin" : "*"
+	},
+	"to" : {
+		"base_url" : "https://localhost",
+		"email_from" : "no-reply@traffic-ops.test",
+		"no_account_found_msg" : "A Traffic Ops user account is required for access. Please contact your Traffic Ops user administrator."
+	},
+	"portal" : {
+		"base_url" : "https://not-a-real-host.test/#!/",
+		"email_from" : "no-reply@traffic-portal.test",
+		"pass_reset_path" : "user",
+		"user_register_path" : "user"
+	},
+	"secrets" : [
+		"blahblah"
+	],
+	"geniso" : {
+		"iso_root_path" : "/opt/traffic_ops/app/public"
+	},
+	"inactivity_timeout" : 60,
+	"smtp" : {
+		"enabled" : false
+	},
+	"InfluxEnabled": false
+}
+EOF
+
+cat <<-EOF >database.conf
+{
+	"description": "Local PostgreSQL database on port 5432",
+	"dbname": "traffic_ops",
+	"hostname": "postgres",
+	"user": "traffic_ops",
+	"password": "twelve",
+	"port": "5432",
+	"ssl": false,
+	"type": "Pg"
+}
+EOF
+
+./traffic_ops_golang --cfg ./cdn.conf --dbcfg ./database.conf >out.log 2>err.log &
+
+cd ../testing/api/v1
+
+makeCFG() {
+	cat <<-EOF >traffic-ops-test.conf
+	{
+		"default": {
+			"logLocations": {
+				"debug": "stderr",
+				"error": "stderr",
+				"event": "stderr",
+				"info": "stderr",
+				"warning": "stderr"
+			},
+			"session": {
+				"timeoutInSecs": 60
+			}
+		},
+		"trafficOps": {
+			"URL": "https://localhost:6443",
+			"password": "twelve",
+			"users": {
+				"disallowed": "disallowed",
+				"operations": "operations",
+				"admin": "admin",
+				"federation": "federation",
+				"portal": "portal",
+				"readOnly": "readOnly",
+				"extension": "extension"
+			}
+		},
+		"trafficOpsDB": {
+			"dbname": "traffic_ops",
+			"description": "Test database to_test",
+			"hostname": "postgres",
+			"password": "twelve",
+			"port": "5432",
+			"type": "Pg",
+			"user": "traffic_ops"
+		},
+		"noPerl": true,
+		"noISO": true
+	}
+	EOF
+}
+
+CODE="0"
+
+makeCFG
+/usr/local/go/bin/go test -v --cfg ./traffic-ops-test.conf
+if [ "$?" -gt "0" ]; then
+	CODE="1"
+fi
+rm traffic-ops-test.conf
+
+cd ../v2
+makeCFG
+ /usr/local/go/bin/go test -v --cfg ./traffic-ops-test.conf
```

Comment:
Since `makeCFG` gives you the same file each time, is `makeCFG` doing anything the 2nd or 3rd time it is being called?

`makeCFG` outputs a file in the directory it is created in, so maybe instead of `go test`, this could be `go test ./v2` with `../traffic-ops-test.conf` so the conf can be in the same place for all 3 tests.

Reply:
I don't know why, but I have had no end of trouble trying to pass anything outside of PWD as the configuration file to those tests. This is actually how I run it locally; I have a separate configuration for each test set, even though they're all the same.

Reply:
It works as-is, so maybe that optimization can come later.

Reply:
I hope someone can figure out why I couldn't get that to work. It was *very* confusing.

Reply:
```
go test ./v3 -test.v -cfg=../conf/traffic-ops-test.conf -fixtures=tc-fixtures.json
```

run from the `api` directory is what I use